github/event_types.go

```diff
@@ -1591,6 +1591,8 @@ type WorkflowRunEvent struct {
 //
 // GitHub API docs: https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#security_advisory
 type SecurityAdvisory struct {
+	CVSs            *AdvisoryCVSs            `json:"cvss,omitempty"`
```

Comment:
I can't do a full review right now, but I noticed that `cvss` is singular (not plural) and is an acronym for "CVS Score" so its capitalization should be "CVSS" . 

The same is true for `AdvisoryCVSS`.